cmd/create.go

```diff
@@ -146,11 +152,42 @@ A full list of component types that can be deployed is available using: 'odo cat
 			os.Exit(1)
 		}
 
+		// If min-memory, max-memory and memory are passed, memory will be ignored as the other 2 have greater precedence.
+		// Emit a message indicating the same
+		if memoryMin != "" && memoryMax != "" && memory != "" {
+			fmt.Println("Flag `memory` will be ignored as `min-memory` and `max-memory` have been passed ")
+		}
+		if (memoryMin == "") != (memoryMax == "") && memory != "" {
+			fmt.Printf("Using memory %s for min and max limits.\n", memory)
+		}
+		if (memoryMin == "") != (memoryMax == "") && memory == "" {
+			fmt.Println("memoryMin should accompany memoryMax or pass memory to use same value for both min and max or try not passing any of them")
+			os.Exit(1)
+		}
+
+		memoryQuantity := util.FetchResourceQunatity(corev1.ResourceMemory, memoryMin, memoryMax, memory)
```

Comment:
This is a minor typo, let's fix it before the PR is merged :)

Reply:
Ack